source/tests/test.ts

```diff
@@ -454,6 +455,14 @@ if (isNode8orHigher) {
 	});*/
 }
 
+if (isNode10orHigher) {
+	test('is.asyncIterable', t => {
+		t.true(m.asyncIterable(Object.create({
+			[Symbol.asyncIterator]: () => {} // tslint:disable-line:no-empty
+		})));
+	});
+}
```

Comment:
Move this below the `is.iterable` test to keep the order. This also needs more assertions, use the same ones as used in the `is.iterable` test.